front/app/components/UI/Button/index.tsx

```diff
@@ -18,11 +18,20 @@ interface ButtonContainerProps extends ComponentLibraryButtonContainerProps {
   'data-testid'?: string;
 }
 
-const ButtonWrapper = ({ linkTo, openLinkInNewTab, ...rest }: Props) => {
+const ButtonWrapper = ({
+  linkTo,
+  openLinkInNewTab,
+  disabled,
+  ...rest
+}: Props) => {
   const locale = useLocale();
   const isExternalLink =
     linkTo && (linkTo.startsWith('http') || linkTo.startsWith('www'));
 
+  if (disabled) {
+    return <Button disabled {...rest} />;
+  }
```

Comment:
Doesn't this need the locale check like the other button lower in the file?

```
  if (!isNilOrError(locale)) {
    return disabled ? (
      <Button disabled {...rest} />
    ) : (
      <Button {...rest} as={link} />
    );
  }

```

Reply:
If yes, not sure why TS is not complaining.

Reply:
I don't think so, the locale is to pass in to the Link to make sure we add it when navigating. If the button is disabled it doesn't do anything and there's no need for the link to have any additional info

Reply:
I think it does in that if there's no locale present, the link won't work because we won't be able to add the locale to the link's destination. It's left over logic from the previous version, I don't think it's ideal but would want to look at it in another ticket/PR if it's not breaking anything

Reply:
Yeah, I think it's a holdover that could easily be removed, but would rather do it elsewhere 👍